packages/aws-cdk/lib/docker.ts

```diff
@@ -95,6 +132,9 @@ async function calculateImageFingerprint(imageId: string) {
   // Metadata that has no bearing on the image contents
   delete manifest.Created;
 
+  // Parent can change when using --cache-from in CI
+  delete manifest.Parent;
```

Comment:
What does `Parent` represent here? Please tell me it has nothing to do with disk images

Reply:
It's the sha256 of the docker's parent image. When using `--cache-from`, the sha256 of the parent gets modified, even for the same final image.

Reply:
Wait, if it's a parent image, and the parent image changes on every build, then we cannot assume that the "current" image is the same either, can we?

Reply:
In this case the sha256 of the layers will change normally. I added this when I saw that for the same digest (on ECR) when using `--cache-from`, `calculateImageFingerprint` returned a different finger print, resulting in 3 tags on the same image (latest, fingerprint1 and fingerprint2). This needs further investigation. The optimal solution may be a mix between the custom finger print calculation and the docker digest. Looking into this.

Reply:
I'm removing this `delete manifest.Parent`, it's a mistake and not needed here. The sha256 of the Parent is also present in the Config.Image key so deleting it in one place only should not have had an effect.